app/models/preserved_copy.rb

```diff
@@ -71,6 +71,15 @@ class PreservedCopy < ApplicationRecord
     # to 0 for nulls, which sorts before 1 for non-nulls, which are then sorted by last_checksum_validation)
   }
 
+  # Send to asynchronous replication pipeline
+  # @raise [RuntimeError] if object is unpersisted or too large (>=~10GB)
+  # @todo reroute to large object pipeline instead of raise
+  def preserve!
```

Comment:
it might be less confusing to rename this method `replicate!`

Reply:
yeah, I was torn between the two names.  `replicate!` is fine by me, though.